front/app/containers/Admin/projects/edit/messages.ts

```diff
@@ -103,7 +103,7 @@ export default defineMessages({
   createSurveyDescription: {
     id: 'app.containers.AdminPage.ProjectEdit.createSurveyDescription',
     defaultMessage:
-      'Embed a Typeform, Google Form, Enalyzer, SurveyXact, Qualtrics, SmartSurvey or Microsoft Forms survey.',
+      'Embed a Typeform, Google Form, Enalyzer, SurveyXact, Qualtrics, SmartSurvey, Snap Survey or Microsoft Forms survey.',
```

Comment:
Changing this here is not enough. With an existing copy - the change needs to happen in Crowdin.

Reply:
For sure. I will update Crowdin as well :+1: We were thinking it should probably match though just to make it clear? Or does this not matter too much?

Reply:
No, it's a good idea :)

Reply:
It was a good idea indeed, because other we would have not included Snap Survey in our [change](https://github.com/CitizenLabDotCo/citizenlab/pull/4540/files#diff-82d52e4cb01b06543d48cdf1d7cc96e41a6877708e9320465318340667effbef).

Maybe you know by now, but just clarifying in case: what needed to happen is change the id ('app.containers.AdminPage.ProjectEdit.createSurveyDescription'), that way it triggers a new string to translate in Crowdin.